build.gradle.kts

```diff
@@ -160,6 +161,34 @@ dependencies {
   testImplementation(libs.testng)
 }
 
+apply(plugin = "org.openapi.generator")
+openApiGenerate {
+  inputSpec = "$rootDir/docs/open-api/openapi.yaml".toString()
+  generatorName = "python"
+  outputDir = "$rootDir/clients/client-python"
+  modelPackage = "dto"
+  modelNameSuffix = "dto"
+  generateApiDocumentation.set(false)
+  generateModelDocumentation.set(false)
+  generateApiTests.set(false)
+  generateModelTests.set(false)
+  packageName.set("com.datastrato.gravitino.client")
+  cleanupOutput.set(true)
+
+  globalProperties.set(
+    mapOf(
+      "apis" to "false",
```

Comment:
One more thing, this will remove a lot of useless files. And the `__init__.py` is removed too. I can't figure out how to solve it for now.

Reply:
We can close `outputClean` and enable `skipOverwrite`, the files that already exist will not overwrite. We can add some custom files such as `__init__.py` for better control.